karma.conf.js

```diff
@@ -11,10 +11,10 @@ var reporters = ['mocha'];
 
 if (runCoverage) {
   webpackConfig = require('./webpack/test-coverage.config');
-  preprocessors.push('coverage');
   reporters.push('coverage');
 
   if (isCI) {
+    preprocessors.push('coverage');
```

Comment:
By pushing this within the `isCI` check doesn't this prevent you from running code coverage locally?